src/common/libterminus/client.c

```diff
@@ -271,9 +271,16 @@ static void pty_client_data (struct flux_pty_client *c, flux_future_t *f)
         llog_error (c, "unpack: %s", error.text);
         return;
     }
-    if (write (STDOUT_FILENO, data, len) < 0) {
-        llog_error (c, "write %zu bytes: %s", len, strerror (errno));
-        return;
+    for (int n = 0; n < len; ) {
+        int ret = write(STDOUT_FILENO, (char *)data + n, len - n);
```

Comment:
`data` doesn't need to be cast to a `char *` - it already is one.

There should be a space between `write` and `(` per RFC 7.

Reply:
Whoops, copy paste error from another spot where data was a void * before the change, thanks!